### PR TITLE
Add fileupload

### PR DESCRIPTION
**Pull request by sirLisko**
**What**  
- Enable CORS on s3 bucket (thanks to @StephenThomson 🙏)
- Improved file upload signature API
- Created a `fileUpload` util
- Created an example page

live example at https://staging-discretionarybusinessgrants.hackney.gov.uk/upload

### CODE REVIEW

File: utils/fileUpload.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+
+const fileUploader = async (file, fileId) => {
+  const { data } = await axios.post('/api/url', {
+    fileId: fileId,
+    fileName: file.name
+  });
+  var formData = new FormData();
```
Comment (jbevan4) [inline utils/fileUpload.js:8]: Should this be a let not a var?

**Comment by jbevan4** [comment]
LGTM only one little question 👍

File: utils/fileUpload.js (hunk 1 above)
Comment (sirLisko) [reply utils/fileUpload.js:8]: Copy and paste for MDN documentation 😅

Comment (jbevan4) [reply utils/fileUpload.js]: don't worry then aha

**Pull request merged**